docs/tutorials/migrate-from-mbed-os-2-to-5.md

```diff
@@ -48,6 +48,20 @@ To determine the success of migration, run:
 
 `mbed compile -m [platform] -t [toolchain]`
 
+The general outline for updating to Mbed OS 5 with the [Mbed Online Compiler](https://os.mbed.com/docs/latest/tools/arm-mbed-online-compiler.html) is:
+
+1. Open your project in the online compiler
+1. Right click on `mbed` and select "Delete..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-delete-mbed.png)</span>
+1. Right click on `mbed-rtos` and select "Delete..."
```

Comment:
@yennster : I didn't find the "mbed-rtos" directory in blinky example for Mbed classic when I import into online compiler from here: https://os.mbed.com/teams/mbed/code/mbed_blinky/  .. you may want to get rid of this line?? or am I missing something?

Reply:
@ashok-rao You're right, Mbed OS 2 blinky doesn't use Mbed RTOS (aka Mbed OS 3). I'll add a clarifying statement saying something like "If your project also uses `mbed-rtos` you'll need to delete this library as well in order to successfully upgrade to Mbed OS 5..."